docs/advanced/middleware.md

```diff
@@ -1,15 +1,16 @@
-In most cases the `dispatcher` in our `store` sends incoming `action` objects directly to the `reducer` to determine what updates should be made to the `state` object tree, but sometimes we want some additional processing to be done on the `action` that our `reducer` cannot provide. Features like delayed processing of an `action`, logging `action` objects dispatched to our `store`, or waiting for the response from a network request before sending our `action` payload to the `reducer` are quite cumbersome within the strict confines of Rodux. We circumvent this problem by introducing custom `middleware` that processes `action` objects before they are passed on to the `reducer`, allowing us to break out of the Rodux paradigm in a controlled and consistent fashion when necessary.
+Most of the time, calling `Store:dispatch` sends incoming `action` objects directly to the `reducer` to determine what updates should be made to the `state` object tree. This is enough for most cases, but some features would be difficult to implement if this was all Rodux provided. For example:
 
-When creating a `store` via [`Store.new`](../api-reference.md#storenew), you can optionally provide `middleware` like so:
+- Delayed processing of an `action`.
+- Logging `action` objects dispatched to our `store`.
+- Performing a network request in response to an `action` and storing the response in the `state` object tree.
 
-!!! info
-	the following examples assume that you've successfully [installed Rodux](installation.md) into `ReplicatedStorage`!
+Rodux has the concept of `middleware` chaining to deal with these sorts of situations.
```

Comment:
Probably remove the word "chaining" here and just go with "the concept of `middleware`"